supplier_app/tests/factory_boy.py

```diff
@@ -49,6 +49,7 @@ class Meta:
     cuit = "1234569"
     comments = "Justificacion falsa"
     payment_type = "Banco"
+    payment_term = 15
```

Comment:
`payment_term` should be one of a list of possible payment term values

Reply:
resolved